gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/TraversalExplanation.java

```diff
@@ -106,7 +107,12 @@ public String prettyPrint(final int maxLineLength) {
         if (maxTraversalColumn < 1)
             throw new IllegalArgumentException("The maximum line length is too small to present the " + TraversalExplanation.class.getSimpleName() + ": " + maxLineLength);
         int largestTraversalColumn = Stream.concat(Stream.of(Pair.with(null, this.traversal)), this.strategyTraversals.stream())
-                .map(Pair::getValue1)
+                .map(new Function<Pair<? extends Object,? extends Traversal.Admin<?,?>>, Object>() {
+                    @Override
+                    public Object apply(Pair<? extends Object, ? extends Traversal.Admin<?, ?>> objects) {
+                        return objects.getValue1();
+                    }
+                })
```

Comment:
This is pretty verbose. What's wrong with the `Pair::getValue1` method reference?